pipenv/environment.py

```diff
@@ -761,6 +768,8 @@ def run_activate_this(self):
                 code = compile(f.read(), activate_this, "exec")
                 exec(code, {"__file__": activate_this})
 
+    EnvironmentActivationResult = namedtuple("EnvironmentActivationResult", ["ok"])
```

Comment:
I think `namedtuple` may be a bit much here considering  we are really just leveraging it to store a boolean--could we just use the boolean value directly?